dev/com.ibm.ws.jaxrs.2.x.webcontainer_fat/.classpath

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="fat/src"/>
-	<classpathentry kind="src" path="test-applications/disablejaxrs11/src"/>
```

Comment:
You will want to rebase to pick up my change since it will conflict with this change.